package.json

```diff
@@ -86,11 +88,12 @@
     "to-single-quotes": "^1.0.2",
     "vow": "~0.4.8",
     "vow-fs": "~0.3.4",
-    "xmlbuilder": "^2.6.1",
-    "htmlparser2": "3.8.3"
+    "xmlbuilder": "^2.6.1"
   },
   "devDependencies": {
     "browserify": "^11.0.0",
+    "chai": "^3.3.0",
+    "chai-string": "^1.1.3",
```

Comment:
They can be replaced with `string`, i suspect that is not a popular use-case in other libraries too, that is why those functions are not included in main dist.

My concern is solely becase `chai-string` is not very popular module, which raise flag for me, since might not be stable enough